packages/codec/lib/errors.ts

```diff
@@ -36,3 +36,31 @@ export class StopDecodingError extends Error {
     this.allowRetry = Boolean(allowRetry);
   }
 }
+
+/**
+ * @hidden
+ */
+export function handleDecodingError(
+  dataType: Format.Types.Type,
+  error: any,
+  strict: boolean = false
+): Format.Errors.ErrorResult {
```

Comment:
I don't think all of these should be positional args: `strict` definitely seems like the sort of thing that's worth making named; I'd even say that `dataType` shouldn't be positional. What do you think of:

```typescript
handleDecodingError(error, { type, strict })
```

(I wouldn't be opposed to `error` being named, also, but there's a good argument for it being positional :)

I know this PR isn't really worth debating minor semantics over, but wondering your thoughts if you considered this.

Reply:
Hm, no, I hadn't considered it; thanks for pointing it out.  I'd rather just leave it as-is, though?  The `dataType` will always be necessary; can't have a return value without it.  I could see `strict` being named if there were also an `allowRetry` option, so that those two could exist on an equal footing, but there isn't, because errors with `allowRetry` set are always thrown manually, not via this pattern (now function).  This function is purely internal -- it's not exported from `Codec` -- so I think it's OK to leave this as-is?  If we want to make `strict`  a keyword argument it wouldn't be breaking or anything.